node/node.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/lightstreams-network/lightchain/database"
 	"github.com/lightstreams-network/lightchain/consensus"
 	"github.com/lightstreams-network/lightchain/log"
-	conAPI "github.com/lightstreams-network/lightchain/consensus/api"
+	consAPI "github.com/lightstreams-network/lightchain/consensus/api"
```

Comment:
[Minor] Better alias `consensusAPI`. `cons` leads to other meanings

Reply:
that's too long. What are you personally confusing cons with? I avoided "con" because that stands normally for connection.

Reply:
I don't think consensusAPI is too long, we used longer name in many other places